shared/react-native/android/app/src/main/AndroidManifest.xml

```diff
@@ -42,7 +42,7 @@
           <data android:mimeType="image/*" />
           <data android:mimeType="video/*" />
           <data android:mimeType="audio/*" />
-          <data android:mimeType="application/pdf" />
+          <data android:mimeType="application/*" />
```

Comment:
@MarcoPolo @cjb I've found myself wanting to upload a zip file before. Please let me know if either of you think this change is too risky.